backend/shork/src/main/kotlin/Settings.kt

```diff
@@ -12,10 +12,22 @@ import software.shonk.interpreter.internal.instruction.Dat
  *   interpreter stops. -1 for no limit
  */
 class Settings(
-    val coreSize: Int,
-    val instructionLimit: Int,
-    val initialInstruction: String,
-    val maximumTicks: Int,
+    /** The size of the core, in number of instructions it will fit. */
+    val coreSize: Int = 8092,
```

Comment:
We should use a more common coresize. KOTH specifies a default coresize of 8000, the "official" standard uses 8192.

Reply:
Yup, I forgot the 1 for 8192